code-challenges.js

```diff
@@ -21,22 +21,78 @@ const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+// Input of boilingPoint should be a temperature value (temp)
+// Output of boilingPoint should be a string that says "x is below the boiling point" or "x is at the boiling point" or "x is above the boiling point" or "x is at the boiling point".
+// Create a funciton called boilingPoint
+// boilingPoint will take in one parameter (temp)
+// Write an if statement stating whether or not the inputted temperature is strictly equal to 212.
+// Write an else if statement stating whether or not the inputted temperature is less than 212.
+// Write an else statment stating that any other inputted temperature has to be greather than 212.
+// Return a string for each condition that is appropriate.
+// Outside of the function, write a console.log to output the expected results on the terminal.
+
+const boilingPoint = (temp) => {
+  if(temp === 212){
+    return `${temp} is at boiling point.`
+  } else if(temp < 212) {
+    return `${temp} is below the boiling point.`
+  } else{
+    return `${temp} is above the boiling point.`
+  }
+}
+console.log(boilingPoint(temp3))
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+// Goal is to add the two arrays (myNumbers1 and myNumbers2) together into one single array and returning the length of the combined array.
+// Create a new variable 'combinedArray' which will be the new array after combining myNumbers1 and myNumbers2.
+// Within the new variable, combinedArray, use the .concat() built-in method to add myNumbers1 and myNumbers2 together.
+// Write a console.log of combinedArray with the extention of .length since our goal is to return the length of the new array.
+
+var combinedArray = myNumbers1.concat(myNumbers2)
+console.log(combinedArray.length)
+
```

Comment:
Nice job. This will get you the right answer but you could also consider chaining the `.length` operator on the end of line 55.